cmake/TestLibXml2ConstErrorStructuredErrorCallback.cmake

```diff
@@ -0,0 +1,91 @@
+# Function: Test_LibXml2_Const_Error_Structured_Error_Callback
+#
+# Test if LibXml2 uses const Error * for structured error callback function.
+#
+# CONST_ERROR_STRUCTURED_ERROR_CALLBACK - true if const Error * is required
+# for structuredErrorCallback.
+#
+
+function(Test_LibXml2_Const_Error_Structured_Error_Callback)
+
```

Comment:
No need for that empty line.

Reply:
[1180414](https://github.com/cellml/libcellml/pull/1274/commits/1180414d58800111c40dd7ae6daff8acd10f736d)